tier0/{{cookiecutter.project_slug}}/README.md

```diff
@@ -1,145 +0,0 @@
-# {{ cookiecutter.project_name }}
-{{ cookiecutter.project_description }}
```

Comment:
I'm personally an advocate for keeping README.md since I think a project description is essential to have in a repository for context. Remy and I were exploring maybe generating a description using code.json. For example:
```
“REPONAME is a LANGUAGE project by CMS that PURPOSE by TEAM for REASON”
```